handlers/facebookapp/facebookapp.go

```diff
@@ -982,7 +982,8 @@ type wacComponent struct {
 }
 
 type wacText struct {
-	Body string `json:"body"`
+	Body       string `json:"body"`
+	PreviewURL bool   `json:"preview_url,omitempty"`
```

Comment:
Why not always have that explicitly set instead of omitting that

Reply:
If it is not sent it will be false by default so I thought of omitting it when I don't need to send it.

Reply:
Do you think it's better to make it explicit?